content/en/wastewater.Rmd

```diff
@@ -401,23 +402,24 @@ p <- reworked_figure(xaxis = "date", yaxis = list(daily_rsv_signal, rsv_7_day),
 p <- layout(p, yaxis = list(range = c(0, 0.0008500), exponentformat="none", tickformat=".5f"))
 p
 ```
-```
+
+\`\`\`
```

Comment:
What is the purpose of this?

Reply:
I have a suspicion this was typed by a cat while I was out of the room